synapse/storage/databases/main/events.py

```diff
@@ -1745,6 +1745,13 @@ def non_null_str_or_none(val: Any) -> Optional[str]:
                 (event.state_key,),
             )
 
+            # The `_get_joined_profile_from_event_id` is immutable, except for the
```

Comment:
The name in this comment doesn't match the cache you're invalidating; copy paste issue?

Reply:
OMG you have no idea how often I got confused between these two.